scalar/README.md

```
@@ -0,0 +1,129 @@
+    FileContent: mySwaggerByteSlice,
+    Path:        "swagger",
+    Title:       "Swagger API Docs",
+}
+
+app.Use(scalar.New(cfg))
+```
+
+### Config
+```go
+type Config struct {
+	// Next defines a function to skip this middleware when returned true.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)